nodeup/pkg/model/network.go

```diff
@@ -31,9 +31,11 @@ type NetworkBuilder struct {
 
 var _ fi.ModelBuilder = &NetworkBuilder{}
 
+// Build is responsible for configuring the network cni
 func (b *NetworkBuilder) Build(c *fi.ModelBuilderContext) error {
 	var assetNames []string
 
+	// @TODO need to clean up the code section isn't not easier to read
```

Comment:
`isn't not` whaaat? :P

Reply:
somewhat ironic given its trying to say it isn't the easiest to read :-)